Scripts/RemediationScripts/Readme.md

```diff
@@ -917,7 +917,7 @@ Yes
 Yes
 
 ___ 
-
+Azure_VirtualMachine_SI_Deploy_GuestConfig_Extension
```

Comment:
Was this added by mistake? Please remove

Reply:
Removed